.github/workflows/dotnet-core.yml

```diff
@@ -12,68 +12,88 @@ jobs:
     runs-on: windows-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
```

Comment:
Duplication of "uses", typo?

Reply:
Not typo - it's invoking the Checkout action twice, once for the main repo and again to get a copy of the RDMP source to build against.